packages/next-swc/Cargo.lock

```diff
@@ -821,7 +841,7 @@ dependencies = [
  "swc_css",
  "swc_ecma_loader",
  "swc_ecma_transforms_testing",
- "swc_ecmascript",
+ "swc_ecmascript 0.125.0",
```

Comment:
Please update `styled_components`

Reply:
`swc_ecmascript 0.125.0 ` means `swc_ecmascript` is included multiple times, and one of them is 0.125.0

Reply:
The next version of `styled_components` requires a dependency of `swc_ecmascript` `^0.129.0`, so upgrading it will also still result in `swc_ecmascript` being included multiple times -- in particular `0.125.0` and `0.129.0` in addition to additional crates being included multiple times relative to if `styled_components` isn't updated. See https://crates.io/crates/styled_components/0.18.0/dependencies